accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -581,40 +579,22 @@ public void testOne()
     private static void run(long seed)
     {
         Duration timeout = Duration.ofMinutes(3);
-        Runnable fn = () -> run(seed, 1000);
-        AsyncResult.Settable<?> promise = AsyncResults.settable();
-        Thread t = new Thread(() -> {
```

Comment:
this timeout logic is now in `TimeoutUtils` as its copy/pasted in 2 places and a 3rd needed it...